pkgs/by-name/op/opener/package.nix

```
@@ -0,0 +1,28 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+}:
+
+buildGoModule rec {
+  pname = "opener";
+  version = "0.1.5";
+
+  src = fetchFromGitHub {
+    owner = "superbrothers";
+    repo = "opener";
+    rev = "refs/tags/v${version}";
```

```suggestion
let 
  version = "0.1.5";
in
buildGoModule {
  pname = "opener";
  inherit version;

  src = fetchFromGitHub {
    owner = "superbrothers";
    repo = "opener";
    rev = "refs/tags/v${version}";
```
I would do it like that, but that's a mostly personal nit. See https://github.com/NixOS/nixpkgs/issues/315337#issuecomment-2138814370 for a comparison.

---

I was taught that you should mostly use it in large files rather than small ones.

---

I'll leave it as, thanks for the heads up